setup.py

```diff
@@ -72,6 +72,7 @@
         "Programming Language :: Python :: 3.7",
```

Comment:
@sitingren remove also python 3.5/3.6 from `setup.py`?

Reply:
That will be done and included in next release. Features in current release (v1.0.5) has been tested in py3.5 and py3.6.